src/main/java/me/arminb/sara/entities/Question.java

```diff
@@ -0,0 +1,89 @@
+package me.arminb.sara.entities;
+
+import org.bson.types.ObjectId;
+
+import java.util.Date;
+
+
+public class Question {
+
+    private ObjectId _id;
+
+    private String title;
+
+    private String content;
+
+    private String[] answers;
```

Comment:
I would make answer a class for its own. The answers of a question can also have comments and voting which can be classes for themselves. I would take a closer look to a stackoverflow question to get a better sense of entities that are present there and their corresponding fields.